sys-process/systemd-cron/systemd-cron-1.5.12-r1.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+PYTHON_COMPAT=( pypy3 python3_{4,5,6} )
+inherit eutils python-single-r1 systemd
+
+DESCRIPTION="systemd units to create timers for cron directories and crontab"
+HOMEPAGE="https://github.com/systemd-cron/systemd-cron/"
+SRC_URI="https://github.com/systemd-cron/${PN}/archive/v${PV}.tar.gz -> systemd-cron-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cron-boot etc-crontab-systemd minutely setgid test yearly"
+
+RDEPEND=">=sys-apps/systemd-217
+	     sys-apps/debianutils
+	     !etc-crontab-systemd? ( !sys-process/dcron )
+	     ${PYTHON_DEPS}
+		 sys-process/cronbase"
+
+DEPEND="sys-process/cronbase
```

Comment:
Throw sys-process/cronbase into a CDEPEND or COMMON_DEPEND variable, and then use that variable in DEPEND and RDEPEND.